rlkit/core/rl_algorithm.py

```diff
@@ -565,7 +450,10 @@ def evaluate(self, epoch):
             # TODO: put these lines into method evaluate_task(idx) that goes into sac.py?
             for _ in range(10):
                 self.infer_posterior(idx)
-                paths += self.eval_sampler.obtain_samples(num_samples=self.max_path_length + 1, deterministic=True, resample=np.inf)
+                # why do we want max_path_length + 1?
+                paths += self.eval_sampler.obtain_samples(num_samples=self.max_path_length + 1,
+                                                          deterministic=True,
```

Comment:
can we change this to `p, _ = ...` and then add `p` to `paths`? I think it's quite opaque to do the zero index - you wouldn't know without jumping to def if there are multiple return values or the return is a list or an array...

Reply:
done 8e25cbe